lib/analyze_hoc_activity_helper.rb

```diff
@@ -63,7 +63,13 @@ def rank_tutorials(end_date)
     memo[key] = {}
     start_date = end_date - num_days.days
     PEGASUS_REPORTING_DB_READONLY.fetch(
-      "SELECT tutorial, #{WEIGHTED_COUNT} #{generate_from_where(start_date, end_date: end_date, tutorial_pixel: false)} GROUP BY tutorial ORDER BY count DESC"
+      "SELECT tutorial, COUNT(*) AS count
+        FROM hoc_learn_activity
+        WHERE (
+          created_at >= '#{start_date.strftime('%Y-%m-%d')}' AND
+          created_at < '#{end_date.strftime('%Y-%m-%d')}'
+        )
+        GROUP BY tutorial ORDER BY count DESC"
```

Comment:
not counting historic tutorials is intentional; we specifically want to count the relative popularity of the tutorials on the `/learn` page. If we were to backfill, we'd want to at most backfill the last few week's worth.